docs/dashboards/custom-chart.md

```diff
@@ -0,0 +1,164 @@
+# Custom Chart
+
+The Custom Chartm Block is an advanced Block that allows you to create custom data visualization by utilizing [Vega-Lite](https://vega.github.io/vega-lite/). If you would like a specific visualization that's not already a Losant Dashboard Block, the Custom Chart Block gives you the ultimate support for a wide array of visualization types.
+
+![Data Table Scatter Plot](../images/workflows/custom-chart-data-table-scatter-plot.png "Data Table Scatter Plot")
+
+## Configuration
+
+Configuring the Custom Chart Block is broken up into three sections ...
+
+### Choosing an Application
+
+First, choose an application that owns the devices and/or the data tables you would like to query.
+
+![Select Application](/images/workflows/custom-chart-select-application.png "Select Application")
+
+### Datasets
+
+![Datasets](/images/workflows/custom-chart-datasets.png "Datasets")
+
+#### Time Series Datasets
+
+Time series datasets provide the value of a single attribute from a selected device over a duration of time at a selected resolution. The values returned from the query will be returned as an array of objects in the form of `{ time: <time>, value: <value> }`. The value returned will reflect the selected attribute's value specified at a specific time.
+
+![Time Series Query](/images/workflows/custom-chart-time-series-query.png "Time Series Query")
+
+#### Guage Datasets
+
+Guage datasets allow you to query a single attribute from a selected device and will return an object in the same form as an individual object from a time series dataset. You can select a duration which will pull all available data that was recorded in the selected duration. If you are pulling any data that is not just the last reported data point, you will need to select an aggregation to perform on the data.
```

Comment:
Could add a note about how this data is similar to the data returned to the Guage Block.

Also, add an example of the JSON payload.